src/front/glsl_new/ast.rs

```diff
@@ -23,9 +23,9 @@ impl Program {
 
 #[derive(Debug)]
 pub enum Profile {
```

Comment:
why do we have that, anyway? We are only interested in GLSL in Vulkan flavor. Does, say, ES profile support Vulkan flavor?

Reply:
You're right, only core profile would actually work. Can't help thinking it would be fantastic if it was somehow possible to support the version used for webgl - to ease developers transitioning from webgl > webgpu, but guessing that is a pipe dream 🤔